Userland/Libraries/LibWeb/Painting/ImagePaintable.cpp

```diff
@@ -62,11 +62,36 @@ void ImagePaintable::paint(PaintContext& context, PaintPhase phase) const
 
     PaintableBox::paint(context, phase);
 
+    auto paint_alt_rect = [&](Gfx::Rect<int> rect, Palette palette) {
+        Gfx::Path top_left;
+        Gfx::Path bottom_right;
+        Gfx::IntRect shrunked = rect.shrunken(2, 2);
```

Comment:
typo: shrunked -> shrunken. We also tend to use `auto` for locals where possible.